### ISSUE TITLE
Cannot pip install git master version of scapy: error: package directory 'scapy/crypto' does not exist

### ISSUE THREAD

**Issue by anadahz** [secdev/scapy#229]
`# pip install --upgrade git+https://github.com/secdev/scapy.git#egg=scapy`
```
Downloading/unpacking scapy from git+https://github.com/secdev/scapy.git
  Cloning https://github.com/secdev/scapy.git to /tmp/pip-build-SBayOx/scapy
  Running setup.py (path:/tmp/pip-build-SBayOx/scapy/setup.py) egg_info for package scapy
    /home/user/ooni-probe/ooni-env/local/lib/python2.7/site-packages/setuptools/dist.py:364: UserWarning: Normalizing '2.3.2-dev' to '2.3.2.dev0'
      normalized_version,

    error: package directory 'scapy/crypto' does not exist
    Complete output from command python setup.py egg_info:
    /home/user/ooni-probe/ooni-env/local/lib/python2.7/site-packages/setuptools/dist.py:364: UserWarning: Normalizing '2.3.2-dev' to '2.3.2.dev0'

  normalized_version,

running egg_info

creating pip-egg-info/scapy.egg-info

writing pip-egg-info/scapy.egg-info/PKG-INFO

writing top-level names to pip-egg-info/scapy.egg-info/top_level.txt

writing dependency_links to pip-egg-info/scapy.egg-info/dependency_links.txt

writing manifest file 'pip-egg-info/scapy.egg-info/SOURCES.txt'

warning: manifest_maker: standard file '-c' not found



error: package directory 'scapy/crypto' does not exist
```
ref: https://github.com/TheTorProject/ooni-probe/issues/573

**Comment by guedou** [comment]
Thanks for reporting this issue.

Does this PR fixes your issue ? https://github.com/secdev/scapy/pull/224

**Comment by guedou** [comment]
Fixed by PR #224.
